submit_cgap/utils.py

```diff
@@ -58,4 +59,21 @@ def json(self):
 
     def raise_for_status(self):
         if self.status_code >= 300:
-            raise Exception("%s raised for status." % self)
+            raise Exception(f"{self} raised for status.")
+
+
+DEBUG_CGAP = environ_bool("DEBUG_CGAP")
+
+
+@contextlib.contextmanager
+def script_catch_errors():
+    try:
+        yield
+        exit(0)
+    except Exception as e:
+        if DEBUG_CGAP:
+            # If debugging, let the error propagate, do not trap it.
+            raise
+        else:
+            show(f"{e.__class__.__name__}: {e}")
```

Comment:
You might consider adding a note here intended for external users telling them to report the error to us, including this error message.

Reply:
Good idea.